assets/scss/modules/mixins/_color.scss

```diff
@@ -0,0 +1,34 @@
+$colours:
+  "ghostwhite" $dp-color-ghostwhite,
+  "lightgrey" $dp-color-lightgrey,
+  "grey" $dp-color-grey,
+  "darkgrey" $dp-color-darkgrey,
+  "lightgreen" $dp-color-lightgreen,
+  "green" $dp-color-green,
+  "darkgreen" $dp-color-darkgreen,
+  "brightness" $dp-color-green-brightness,
+  "sylver" $dp-color-sylver,
+  "snow" $dp-color-snow,
+  "lightyellow" $dp-color-lightyellow,
+  "lemonyellow" $dp-color-lemonyellow,
+  "yellow" $dp-color-yellow,
+  "darkyellow" $dp-color-darkyellow,
+  "brightness" $dp-color-yellow-brightness,
+  "lightred" $dp-color-lightred,
+  "red" $dp-color-red,
+  "darkred" $dp-color-darkred,
+  "white" $dp-color-white,
+  "border-grey" $dp-border-grey,
+  "default" $dp-color-default,
+  "lightbrown" $dp-color-lightbrown,
+  "brown" $dp-color-brown,
+  "brown-hover" $dp-color-brown--hover;
+
+  @each $i in $colours{
```

Comment:
Nice!

Reply:
This is pretty nice, @IgnacioRodrigues . GJ buddy 💪